src/transformers/models/wav2vec2/tokenization_wav2vec2.py

```diff
@@ -351,7 +351,7 @@ def _get_word_offsets(
                     word = char
 
             last_state = state
-        if state == "WORD":
+        if last_state == "WORD":
```

Comment:
This is not 100% necessary, but it's more "correct" since `state` represents something in the loop, whereas this is defined outside of the loop.

Reply:
ok for me!